src/job/jobHandler.ts

```diff
@@ -274,11 +274,18 @@ export abstract class JobHandler {
     }
     const snootyFrontEndVars = {
       GATSBY_BASE_URL: this._config.get<string>('gatsbyBaseUrl'),
+      GATSBY_TEST_EMBED_VERSIONS: this._config.get<boolean>('gatsbyTestEmbedVersions'),
     };
+    this._logger.info(this._currJob._id, 'check snootyFrontEndVars');
+    this._logger.info(this._currJob._id, JSON.stringify(snootyFrontEndVars));
```

Comment:
I like this - but we should probably add a comment for future developers that they need to be *extra* judicious about not logging anything which might be a secret here.

Reply:
debugging remnants for [front end fix](https://github.com/mongodb/snooty/pull/751) 😔